packages/kolibri-tools/yarn-error.log

```diff
@@ -0,0 +1,14515 @@
+Arguments: 
+  /home/richard/.nvm/versions/node/v10.12.0/bin/node /home/richard/.yarn/bin/yarn.js add webpack-file-loader
+
+PATH: 
+  /home/richard/.nvm/versions/node/v10.12.0/bin:/home/richard/.virtualenvs/kolibripy2/bin:/home/richard/.yarn/bin:/home/richard/.config/yarn/global/node_modules/.bin:/usr/local/bin:/home/richard/.cargo/bin:/home/richard/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/usr/lib/jvm/java-8-oracle/bin:/usr/lib/jvm/java-8-oracle/db/bin:/usr/lib/jvm/java-8-oracle/jre/bin
```

Comment:
whoops - looks like this error log was committed by accident. My current PR has some yarn issues, too. I'll fix those those up once this is updated

Reply:
nevermind - addressed it here: https://github.com/rtibbles/kolibri/pull/16/commits/e77b3bd0b7aec5dd4ed35a6627e68bac853a55cb